docs/_templates/versions.html

```diff
@@ -1,8 +1,4 @@
 <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="versions">
-  <span class="rst-current-version" data-toggle="rst-current-version">
-    <span class="rst-current-version-label">{{ version_label }}</span>
-    <span class="rst-versions-dropdown-icon"></span>
-  </span>
   <div class="rst-other-versions">
```

Comment:
I think we want to keep this right? It's what was added back in https://github.com/Qiskit/qiskit-ibm-runtime/pull/612

Reply:
Oh yeah; it didn't show up as a merge conflict, so I didn't notice it was missing.  thanks!